lib/utilities/aclUtils.js

```diff
@@ -117,7 +117,7 @@ aclUtils.parseAclXml = function parseAclXml(toBeParsed, log, next) {
 
         if (!result.AccessControlPolicy.Owner
```

Comment:
does this make more sense? @rahulreddy @alexandre-merle 
https://github.com/scality/S3/pull/379/commits/12dfa4133253f334e0f2e3cef3ff4c1834cfea15